python-client/giskard/core/suite.py

```diff
@@ -224,11 +225,25 @@ def run(self, verbose: bool = True, **suite_run_args):
             raise ValueError(f"Missing {len(undefined_params)} required parameters: {undefined_params}")
 
         for test_partial in self.tests:
-            test_params = self.create_test_params(test_partial, suite_run_args)
-            result = test_partial.giskard_test.get_builder()(**test_params).execute()
-            res.append((test_partial.test_name, result))
-            if verbose:
-                print("""Executed '{0}' with arguments {1}: {2}""".format(test_partial.test_name, test_params, result))
+            try:
+                test_params = self.create_test_params(test_partial, suite_run_args)
+                result = test_partial.giskard_test.get_builder()(**test_params).execute()
+                res.append((test_partial.test_name, result))
+                if verbose:
+                    print(
+                        """Executed '{0}' with arguments {1}: {2}""".format(test_partial.test_name, test_params, result)
+                    )
+            except BaseException as e:  # noqa SONAR
+                error = traceback.format_exc()
+                logging.exception("An error happened during test execution")
```

Comment:
```suggestion
                logging.exception("An error happened during test execution", e)
```

Reply:
Well actually when logging exception in python we don't need to pass the exception and it will log it automatically, I got an error if I pass as in the code snippet.

https://docs.python.org/3/library/logging.html#logging.Logger.exception